components/servo/servo.go

```diff
@@ -31,6 +31,30 @@ const SubtypeName = "servo"
 var API = resource.APINamespaceRDK.WithComponentType(SubtypeName)
 
 // A Servo represents a physical servo connected to a board.
+//
+// Move example:
+//    // Move the servo from its origin to the desired angle of 30 degrees.
+//    myServoComponent.Move(context.Background(), 30, nil)
+//
+// Position example:
+//    // Get the current set angle of the servo.
+//    pos1, err := myServoComponent.Position(context.Background(), nil)
+//
+//    // Move the servo from its origin to the desired angle of 20 degrees.
+//    myServoComponent.Move(context.Background(), 20, nil)
+//
+//    // Get the current set angle of the servo.
+//    pos2, err := myServoComponent.Position(context.Background(), nil)
+//
+//    logger.Info("Position 1: ", pos1)
+//    logger.Info("Position 2: ", pos2)
+//
+// Stop 
+//    // Move the servo from its origin to the desired angle of 10 degrees.
+//    myServoComponent.Move(context.Background(), 10, nil)
+//
+//    // Stop the servo. It is assumed that the servo stops moving immediately.
+//    myServoComponent.Stop(context.Background(), nil)
```

Comment:
At first, I was surprised that this was here, given that it's not obviously mentioned in the `Servo` interface. On second glance, it comes from within `resource.Actuator`. Is it worth mentioning that in the comments up here? Maybe not; maybe leave as-is.

Reply:
Yes, I was wondering about that and asked a minute ago in team-sdk (which was maybe not the place to ask--should've asked bucket!). We have it in the docs under servo so I thought it should be included but you're right I should put something to clarify here!